config/crd/bases/dataworkflowservices.github.io_clientmounts.yaml

```diff
@@ -278,16 +278,8 @@ spec:
     subresources:
       status: {}
   - additionalPrinterColumns:
-    - description: mounted/unmounted
```

Comment:
Ready tells us something, but without the desired state we don't know if they are mounted or not. Don't you think we need these?

Reply:
I'll add that back. The current state is tricking since we'd need to roll that up, so that's why I took these out. But desired state is straight forward.